pages/_app.tsx

```diff
@@ -1,10 +1,12 @@
 import type { AppProps } from 'next/app'
 import Layout from 'components/Layout'
+import CreateNewUserAccount from './CreateNewUserAccount/CreateNewUserAccount'
 
 function MyApp({ Component, pageProps }: AppProps) {
   return (
     <Layout>
       <Component {...pageProps} />
+      <CreateNewUserAccount />
```

Comment:
think i already mentioned this previously. you do not need to modify this file. ur work will be at the path of u specified in the pages folder. https://nextjs.org/docs/routing/introduction